### PR TITLE
Documented how to add or remove values of multi-valued attributes

### PR DESCRIPTION
**Pull request by jean-gui**
Related to https://github.com/symfony/symfony/pull/21856

### CODE REVIEW

File: components/ldap.rst [hunk 1]
```diff
@@ -129,6 +129,10 @@ delete existing ones::
     $entry = $result[0];
     $entry->setAttribute('email', array('fabpot@symfony.com'));
     $entityManager->update($entry);
+    
+    // Adding or removing values to a multi-valued attribute is more efficient than using update()
+    $entityManager->addAttributeValues($entry, 'telephoneNumber', ['+1.111.222.3333', '+1.222.333.4444']);
```
Comment (xabbuh) [inline components/ldap.rst]: For consistency please us the long array notation here.

**Comment by xabbuh** [comment]
Should we also talk about the new `removeAttributeValues()` method?

**Comment by xabbuh** [comment]
I would also add a `versionadded` directive below to explain that these methods are new:

```rst
.. versionadded:: 4.1
    The ``addAttributeValues()`` and ``removeAttributeValues()`` methods
    were introduced in Symfony 4.1.
```

File: components/ldap.rst [hunk 2]
```diff
@@ -129,8 +129,15 @@ delete existing ones::
...
+    // Adding or removing values to a multi-valued attribute is more efficient than using update()
+    $entityManager->addAttributeValues($entry, 'telephoneNumber', array('+1.111.222.3333', '+1.222.333.4444'));
+    $entityManager->removeAttributeValues($entry, 'telephoneNumber', array('+1.111.222.3333', '+1.222.333.4444'));
 
     // Removing an existing entry
     $entityManager->remove(new Entry('cn=Test User,dc=symfony,dc=com'));
 
+.. versionadded:: 4.1
+    The ``addAttributeValues()`` and ``removeAttributeValues()`` methods
+    were introduced in Symfony 4.1.
```
[the remark this answers is not in the document]
Comment (jean-gui) [reply components/ldap.rst]: I've added it.

**Review by xabbuh — APPROVED** [review]

**Comment by javiereguiluz** [comment]
@jean-gui thanks for adding this missing docs. It was a nice first contribution to Symfony Docs!

**Pull request merged**